cypress/integration/home_page_spec.js

```diff
@@ -8,6 +8,20 @@ describe('Not Fur-gotten Home Page User Flow', () => {
           'x-gql-operation-name': 'getAllPets',
         }
       },
+      {
```

Comment:
These fixtures. There are a LOT! Glad we were able to work through them the other day, so they weren't quite so painful today. Thanks for being the driver on this one.